cli/tools/test/mod.rs

```diff
@@ -640,6 +640,7 @@ async fn configure_main_worker(
         stdout: StdioPipe::file(worker_sender.stdout),
         stderr: StdioPipe::file(worker_sender.stderr),
       },
+      None,
```

Comment:
Note to self: rewrite `create_custom_worker` to accept option bug instead of so many arguments